README.md

```diff
@@ -159,13 +161,13 @@ The scripts are divided into two sections: "song scripts" and "project scripts".
 
 #### `yarn script build` (alt. `yarn build`)
 
-Builds files. Can optionally provide a date in the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601/) format to mark as date of most recent update.
+Builds files. Can optionally provide a date in the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601/) format to mark as the date of the most recent update.
```

Comment:
```suggestion
Builds files. Can optionally provide a date in the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601/) format to mark as the date of the most recent update. If a date is not provided it will default to today's date.
```

Reply:
Resolved with fixup